pokecli.py

```diff
@@ -46,21 +46,24 @@
 
 def init_config():
     parser = argparse.ArgumentParser()
-    config_file = "config.json"
-    release_config_json = "release_config.json"
+
     web_dir = "web"
+    _root_dir = os.path.dirname(os.path.realpath(__file__))
+    _config_dir = os.path.join(_root_dir, "config.json")
+    _release_config_dir = os.path.join(_root_dir, "release_config.json")
```

Comment:
dev branch doesn't use release_config.json anymore and config files are in configs/, not root